sdk/ai/azure-ai-projects/README.md

```diff
@@ -327,6 +330,95 @@ agent = project_client.agents.create_agent(
 
 <!-- END SNIPPET -->
 
+#### Create Agent with Enterprise File Search
+
+We can upload file to Azure as it is shown in the example, or use the existing Blob. In the code below we demonstrate how this can be achieved. First we are upload file to azure and create `VectorStoreDataSource`, which then is used to create vector store. This vector sore is then given to the `FileSearchTool` constructor.
+
+<!-- SNIPPET:sample_agents_enterprise_file_search.upload_file_and_create_agent_with_file_search -->
+
+```python
+# We will upload the local file to Azure and will use it for vector store creation.
+_, asset_uri = project_client.upload_file("./product_info_1.md")
+
+# create a vector store with no file and wait for it to be processed
```

Comment:
Please be consistent: Code comments start with capital letter, as in regular text.

Reply:
Fixed.